src/Stateless/StateMachine.cs

```diff
@@ -18,6 +28,7 @@ public partial class StateMachine<TState, TTrigger>
         readonly Action<TState> _stateMutator;
         UnhandledTriggerAction _unhandledTriggerAction;
         OnTransitionedEvent _onTransitionedEvent;
+        private readonly FiringMode _firingMode;
```

Comment:
Nitpick: explicit `private` is inconsistent with the lines above.

Reply:
I'll add private to everything :-D